Northwind/Templates/EFCore/DbContext.t4

```diff
@@ -18,11 +13,9 @@
 <#@ parameter name="Model" type="Microsoft.EntityFrameworkCore.Metadata.IModel" #>
 <#@ parameter name="Options" type="Microsoft.EntityFrameworkCore.Scaffolding.ModelCodeGenerationOptions" #>
 <#@ import namespace="Microsoft.EntityFrameworkCore.Metadata.Internal" #>
-<#@ import namespace="Microsoft.EntityFrameworkCore.Storage" #>
 <#
     var services = (IServiceProvider)Host;
     var providerCode = services.GetRequiredService<IProviderConfigurationCodeGenerator>();
-    var annotationCode = services.GetRequiredService<IAnnotationCodeGenerator>();
```

Comment:
This can be used to generate Fluent API calls, but I liked how clear your templates were too much to ruin them. 😉